sympy/core/tests/test_var.py

```diff
@@ -1,83 +1,87 @@
 # Tests for var are in their own file, because var pollutes global namespace.
```

Comment:
The comment at the beginning isn't needed anymore as now all the calls to `var` are inside `exec` and don't pollute the global namespace